Aem64/src/test/java/com/cognifide/qa/bb/aem/tests/login/AuthorizationTest.java

```diff
@@ -22,26 +24,27 @@
 public class AuthorizationTest {
 
   @Inject
-  private AemAuthenticationController aemAuthenticationController;
+  private Controller controller;
 
   @Inject
   private BobcatPageFactory bobcatPageFactory;
 
   @Test
   @Story("Login to AEM and open Test page")
   @Description("Login to author instance and open test page")
-  public void loginTest() {
+  public void loginTest() throws ActionException {
     TestPage testpage = bobcatPageFactory.create("/content/we-retail/us/en.html", TestPage.class);
-    aemAuthenticationController.login();
-    assertTrue(((TestPage) testpage.open()).isDisplayed());
+    controller.execute(Actions.Login.LOG_IN);
+    assertTrue(testpage.open().isDisplayed());
   }
 
   @Test
   @Story("Login to AEM and open Test page")
   @Description("Login to author instance and open test page in one step")
-  public void loginAndOpenTest() {
+  public void loginAndOpenTest() throws ActionException {
     TestPage testpage = bobcatPageFactory.create("/content/we-retail/us/en.html", TestPage.class);
-    aemAuthenticationController.login(testpage);
+    controller.execute(Actions.Login.LOG_IN);
```

Comment:
Without removed login with open action (which I think should be back) it is just a copy of code